Docs/Specifics/Vision.md

```diff
@@ -49,27 +51,40 @@ If we also have pose estimates from other tags that are less ambiguous, we can t
 Having multiple cameras also helps with this, as another camera at a different angle could generate a more accurate pose with the same tag.
 Comparing a vision pose against odometry can also rule out incorrect poses.
 
+There are several different strategies to pick which pose we want to use.
+One of these is lowest ambiguity.
+This finds the pose corresponding to the vision result with the lowest reprojection error (a measurement of the difference between a measured 3D point and its projected 2D point in the camera frame).
+This is useful when we only have one tag in frame that we can use to differentiate between these possible poses.
+Another is to choose the pose closest to some other known pose, such as the camera's pose or the last calculated pose.
+Historically in FRC, it's been unlikely that a robot would be above the ground or outside the field, so this can eliminate such poses that are further than the robot could have traveled in the time since the last measurement.
+Lastly, the best way to take advantage of multiple tags is by using the multitag PNP (Perspective-n-Point) strategy.
+This maps the 3D points of visible AprilTags to 2D points to estimate the field-to-camera transformation.
```

Comment:
this doesnt really mean anything to a newbie, and isnt really a useful explanation. instead include something about how it fuses the poses from multiple tags